libraries/math/package.json

```diff
@@ -36,6 +36,7 @@
     "@minecraft/core-build-tasks": "*",
     "@minecraft/tsconfig": "*",
     "just-scripts": "^2.2.1",
+    "prettier": "^2.8.2",
```

Comment:
Just to make sure I understand this correctly, to make it so the lint task uses prettier you only need to add prettier as a dev dependency? If prettier isn't a dev dependency does the lint task normally fail?

Reply:
It does not fail, but it is skipped:
![image](https://github.com/Mojang/minecraft-scripting-libraries/assets/93944312/364b467a-334a-4f57-921b-077b9b3dc85d)

Reply:
huh, cool!